website/templates/profile/account.mako

```diff
@@ -66,12 +65,15 @@
                                 <tr>
                                     <td colspan="3">
                                         <form data-bind="submit: addEmail">
+                                        <p>To merge an existing OSF account with this one or to be able to log in under multiple email addresses, add your alternate email address in the field below.</p>
```

Comment:
Possible revision:

> To merge an existing account with this one or log in with multiple email addresses, add an alternate email address  below.

Although, perhaps we don't need to mention "merging" at all? As a user, I would assume adding an alternate email that is already associated with OSF account would get merged. Also, I believe the messaging that follows this step makes it clear that a merge is happening.

If that assumption holds, then this text might not even be necessary.

Pinging @erinmayhood for review.